apps/info/static/admin/info/js/FestivalTeamFooter.js

```diff
@@ -1,27 +1,33 @@
-// Show/hide the boolean field `is_pr_manager`.
-// The field has to be extended in admin with CSS class `depended_on_team`.
+// Show/hide the field `data_manager`.
+// The field has to be extended in admin with CSS class `form-row field-data_manager`.
 //
 // The logic:
-//  - show the field if selected team type is `art`
-//  - set "checked" to `false` and hide the field if other partner type is selected
+//  - show the field if boolean field `is_pr_manager` is `true`
 
 jQuery(document).ready(function ($) {
-    let teamTypeSelectField = $("#id_team");
-    let divDependedOnTeamType = $(".depended_on_team_type");
+    let idIsPrManager = $("#id_is_pr_manager");
+    let divHideDatePrManager = $("#artteammember_form > div > fieldset:nth-child(2) > div.form-row.field-data_manager");
 
-    function toggleDivDependedOnTeamType(teamType) {
-        if (teamType === "art") {
-            divDependedOnTeamType.slideDown();
-        } else {
-        divDependedOnTeamType.slideUp();
-        }
+    // exit from function if there isn't selector
+    if (idIsPrManager.length === 0) {
```

Comment:
Всегда старайся сокращать код, тем более в проверках.
```suggestion
    if (!idIsPrManager.length) {
```